ext/exif/exif.stub.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/** @return string|false */
+function exif_tagname(int $index) {}
+
+/** @return array|false */
+function exif_read_data($filename, ?string $sections_needed = null, bool $sub_arrays = false, bool $read_thumbnail = false) {}
+
+/** @return string|false */
+function exif_thumbnail($filename, int &$width, int &$height, int &$imagetype) {}
```

Comment:
These reference arguments shouldn't have types, because the input type can be arbitrary.

Reply:
How arbitrary it should be? I mean, in my opinion I think that I shouldn't be able to pass an array at *width*  or *height*

Reply:
@Arkanius, it doesn't matter how it *should* behave, but rather how it *does* behave. Changing the behavior is another story.